ui/xui/compat.cc

```diff
@@ -206,14 +206,22 @@ void CompatibilityReporter::Draw()
     ImGui::SetCursorPosX(ImGui::GetWindowWidth()-(120+10)*g_viewport_mgr.m_scale);
 
     ImGui::SetItemDefaultFocus();
+    if (g_config.perf.override_clockspeed) {
+        ImGui::PushItemFlag(ImGuiItemFlags_Disabled, true);
+        ImGui::PushStyleVar(ImGuiStyleVar_Alpha, ImGui::GetStyle().Alpha * 0.5f);
+    }
```

Comment:
Small change so that the text and button are aligned:

```suggestion
    if (g_config.perf.override_clockspeed) {
        ImGui::PushItemFlag(ImGuiItemFlags_Disabled, true);
        ImGui::PushStyleVar(ImGuiStyleVar_Alpha, ImGui::GetStyle().Alpha * 0.5f);

        ImGui::Text("Reports cannot be made while using altered CPU clock speeds");
        ImGui::SameLine();
    }

    ImGui::SetCursorPosX(ImGui::GetWindowWidth()-(120+10)*g_viewport_mgr.m_scale);

    ImGui::SetItemDefaultFocus();
```

and then remove the `ImGui::Text(...)` on line 221.